lib/shared/addon/components/cluster-driver/driver-googlegke/template.hbs

```diff
@@ -706,12 +706,23 @@
         </div>
         {{#if config.enableNodepoolAutoscaling}}
           <div class="col span-4">
-            <label class="acc-label">{{t "clusterNew.googlegke.minNodeCount.label"}}</label>
-            {{input-number min=1 value=config.minNodeCount max=5}}
+            <label class="acc-label">
```

Comment:
`for="..."` & id on the input please

Reply:
sorry, I don't know why I missed again :(